mqtt-bridge/src/main/java/io/strimiz/strimzi_mqtt_bridge/Main.java

```diff
@@ -0,0 +1,20 @@
+package io.strimiz.strimzi_mqtt_bridge;
+
+import io.netty.channel.ChannelOption;
+import io.netty.channel.nio.NioEventLoopGroup;
+import io.strimiz.strimzi_mqtt_bridge.core.MqttServer;
+
+public class Main {
+    static final int PORT = Integer.parseInt(System.getProperty("port", "1883"));
```

Comment:
You should start to think about a more complete mechanism to provide full configuration.
I would assume that the MQTT `port` is just one of them but at some point we would also allow the users to configure the Kafka producer part of the bridge.
See for example how it's done today in the Strimzi HTTP bridge as a reference.
Maybe not something to add to this PR of course.

Reply:
Yes, I am keeping that in mind.
The same thing holds for the javadoc in classes.